src/grandorgue/dialogs/settings/GOSettingsOrgans.cpp

```diff
@@ -285,9 +282,38 @@ GOSettingsOrgans::PackageSlotSet GOSettingsOrgans::GetUsedPackages(
   return packagesUsed;
 }
 
-bool GOSettingsOrgans::TransferDataToWindow() {
-  m_config.AddOrgansFromArchives();
+void GOSettingsOrgans::DisplayMidiCell(unsigned rowN, GOOrgan *pOrgan) {
+  m_GridOrgans->SetCellValue(
+    rowN,
+    GRID_COL_MIDI,
+    pOrgan->GetMIDIReceiver().GetEventCount() > 0 ? _("Yes") : _("No"));
+}
 
+void GOSettingsOrgans::DisplayPathCell(unsigned rowN, const wxString &path) {
+  m_GridOrgans->SetCellValue(rowN, GRID_COL_PATH, path);
+}
+
+void GOSettingsOrgans::FillGridRow(unsigned rowN, OrganSlot &organSlot) {
+  GOOrgan *o = organSlot.p_CurrentOrgan;
+  wxString title = o->GetChurchName();
+
+  if (!o->IsUsable(m_config))
+    title = _("MISSING - ") + title;
+
+  // m_OrganSlotPtrsByGridLine has at least rowN+1 elements
+  for (unsigned i = m_OrganSlotPtrsByGridLine.size(); i <= rowN; i++)
+    m_OrganSlotPtrsByGridLine.push_back(nullptr);
+
+  m_OrganSlotPtrsByGridLine[rowN] = &organSlot;
+
+  m_GridOrgans->SetCellValue(rowN, GRID_COL_NAME, title);
+  DisplayMidiCell(rowN, o);
+  DisplayPathCell(rowN, organSlot.m_CurrentPath);
+  m_GridOrgans->SetCellAlignment(
+    rowN, GRID_COL_PATH, wxALIGN_RIGHT, wxALIGN_TOP);
```

Comment:
I suggest changing the wxALIGN_RIGHT to wxALIGN_LEFT or simply removing the whole statement for now.